### ISSUE TITLE
Examination of firmadyne scraper

### ISSUE THREAD

**Issue by FendVario1** [amosproj/amos2022ss01-firmware-downloader#1]
## User story
1. As a Developer
2. I want / need an overview of the firmadyne scraper (https://github.com/firmadyne/scraper).
3. So that it is easily possible to get an overview of its functionality.

## Acceptance criteria
* Detailed list of every "surprising" solution
* Overview of general program sequence
* What the crawler is supposed to do & whether this is still working
* All of the above in a document in this repository

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
